spec/services/collectors/daily_counts_stats_collector_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+describe Collectors::DailyCountsStatsCollector do
+  def random_time_today
```

Comment:
nitpick: this is really more `random_time_of_day`, since no day value is inherent. 
```suggestion
  def random_time_of_day
```